docs/spark-connector/spark-catalog-jdbc.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Spark connector JDBC catalog"
+slug: /spark-connector/spark-catalog-jdbc
+keyword: spark connector jdbc catalog
+license: "This software is licensed under the Apache License version 2."
+---
+
+The Apache Gravitino Spark connector offers the capability to read JDBC tables, with the metadata managed by the Gravitino server. To enable the use of the JDBC catalog within the Spark connector, you must download the jdbc driver jar which you used to Spark classpath.
+
+## Capabilities
```

Comment:
The current implement could support Mysql&PG natively, for OB which is compatible with Mysql Dialects could use Mysql driver and Mysql Dialects as a trackoff way. But Doris seems we could only do simple SQL because it may doesn't support Mysql Dialects.

Reply:
I don't know much about Doris. In theory, it can support all JDBC dialects that spark natively supports, but Doris seem to not be among them. Maybe I should mark that Doris is not supported here?

Reply:
Yes, this is the purpose of capabilities